basics/print_data.php

```diff
@@ -0,0 +1,22 @@
+<?php
+require 'email_validation.php';
+function show_data($fullName,$email_address,$number,$image,$marks): void
+{
+    echo "\n<img style='height: 200px' src='{$image}' alt='Uploaded Image'>";
+    echo "<h1>Hello $fullName</h1>";
+    echo "<h2>Subject Marks</h2>";
+    echo "<table border='1px'>";
+    echo "<tr><th>Subject</th><th>Mark</th></tr>";
```

Comment:
Why are we using echo to output HTML?
https://www.geeksforgeeks.org/how-to-use-php-in-html/